### ISSUE TITLE
How to delete a cached item completely?

### ISSUE THREAD

**Issue by zicodeng** [apollographql/apollo-client#5476]
Hi, I want to know how to delete a cached item completely?

E.g. I have a cached item `user`, is there anyway I can completely delete this item (entry) from Apollo cache, not just setting `user = null`, but `delete cache.user`?

Thanks

**Comment by dylanwulf** [comment]
When apollo-client v3 releases, there will be a way to delete specific items from the cache

**Comment by zicodeng** [comment]
@dylanwulf when would that be?

**Comment by dylanwulf** [comment]
Yeah their goal is to have a release candidate ready for the graphql summit at the end of Oct. The general release date is still unknown but hopefully will be soon after

**Comment by hwillson** [comment]
See https://www.apollographql.com/docs/react/caching/garbage-collection/ - thanks!
